app/models/case_information.rb

```diff
@@ -41,6 +41,10 @@ class CaseInformation < ApplicationRecord
 
   before_validation :set_probation_service
 
+  def nps?
+    case_allocation == NPS
+  end
```

Comment:
We have an almost identical method on the `OffenderBase` object, too.

`offender.nps_case?`

If you think the CaseInfo record is the right place for it (it probably is..), then it might be worth delegating `OffenderBase.nps_case?` to `CaseInformation.nps_case?` to keep things DRY.